applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -196,8 +225,14 @@ static struct bt_audio_broadcast_sink_cb broadcast_sink_cbs = { .scan_recv = sca
 								.base_recv = base_recv_cb,
 								.syncable = syncable_cb };
 
-int le_audio_config_get(void)
+int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate)
 {
+	int frames_per_sec = 1000000 / bt_codec_cfg_get_frame_duration_us(streams[0].codec);
+	int bits_per_frame = bt_codec_cfg_get_octets_per_frame(streams[0].codec) * 8;
+
+	*sampling_rate = bt_codec_cfg_get_freq(streams[0].codec);
```

Comment:
Using hardcoded streams[0] works now since we only use one stream, but we should improve on this in the future

Reply:
Yeah, we need to discuss how the interface should be there. Would the application know what stream to get config for? Can we have two streams coming in at the same time realistically?